static/templates/delete_message_modal.hbs

```diff
@@ -0,0 +1,3 @@
+<p><strong>{{t "Are you sure you want to permanently delete this message?"}}</strong></p>
+<p>{{t "Deleting a message removes it for everyone."}}</p>
+<div id="delete-message-error"></div>
```

Comment:
I think this is fine for this PR, but an error element probably would be better to be a `confirm_dialog` feature like the loading spinner, long-term.

Reply:
Done.